frontend/views/pages/GroupDashboard.vue

```diff
@@ -114,20 +114,25 @@ export default ({
     humanDate,
     handleIncomeClick (e) {
       sbp('okTurtles.events/emit', OPEN_MODAL, 'IncomeDetails')
+
+      // firstly, make sure the listener is attached to 'GROUP_PROFILE_UPDATE' more than twice.
```

Comment:
a typo in this comment 'is attached to' -> is '**not**' attached to

Reply:
@taoeffect 
Btw, I've got a small proposal here.

while working on updating this part, I felt it would've been great if there was a method in `@okTurtles/events` library that checks if a handler has already been attached to a particular event. something that looks like this.
```
if (!sbp('okTurtles.events/hasHandler', GROUP_PROFILE_UPDATE, this.closeBanner))
  sbp('okTurtles.events/once', GROUP_PROFILE_UPDATE, this.closeBanner)

```

I went ahead and tried to write those code myself that maybe can be added to [okTurtles.events/index.js ](https://github.com/okTurtles/okTurtles.events/blob/master/index.js)

```
'okTurtles.events/hasHandler': function (event, handler) {
    const handlerList = sbp('okTurtles.data/get', listenKey(event))

    return handlerList ? handlerList.includes(handler) : false
  }

```

Do you think it can be a useful addition to the library?